### DISCUSSION TITLE
How to change the line length to 100 in Black extension in my .vimrc

### DISCUSSION THREAD

**Discussion by imflash217** [w0rp/ale#3966] [Q&A - Ask for help with problems]
Hi,
How to change the line length to `100` in `black` extension in my `.vimrc`?
Thanks

**Comment by hsanson** [comment]
You do not configure the linters/fixers with .vimrc.  You must use whatever configuration methods the tool provides:

  - https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file
